jamon-parent/jamon-examples/src/main/java/org/wicketstuff/jamon/example/HomePage.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wicketstuff.jamon.example;
+
+import org.apache.wicket.markup.html.WebPage;
+import org.apache.wicket.markup.html.link.Link;
+import org.apache.wicket.request.mapper.parameter.PageParameters;
+import org.wicketstuff.jamon.component.JamonAdminPage;
+
+/**
+ * Dummy HomePage used by the {@link JamonWebApplication} in tests.
+ * 
+ * @author lars
+ *
+ */
+@SuppressWarnings("serial")
+public class HomePage extends WebPage
+{
+	public HomePage()
+	{
+		add(new Link<Void>("toStatisticsPage")
+		{
+
+			@Override
+			public void onClick()
+			{
+				setResponsePage(
+					new JamonAdminPage(new PageParameters().set(JamonAdminPage.PARAM_ITEMS, 100)));
+			}
+		});
+		add(new Link<Void>("toAjaxPage")
+		{
+
+			@Override
+			public void onClick()
+			{
+				setResponsePage(new AjaxPage());

```

Comment:
Prefer `BookmarkablePageLink` instead of `Link+setResponsePage()` if this is the only thing it does. 
It saves few HTTP redirects.